Examples/Sources/MacroExamples/Implementation/ComplexMacros/DictionaryIndirectionMacro.swift

```diff
@@ -21,10 +21,9 @@ extension DictionaryStorageMacro: MemberMacro {
     providingMembersOf declaration: some DeclGroupSyntax,
     in context: some MacroExpansionContext
   ) throws -> [DeclSyntax] {
-    let storage: DeclSyntax = "var _storage: [String: Any] = [:]"
-    return [
-      storage.with(\.leadingTrivia, [.newlines(1), .spaces(2)])
-    ]
+    var storage: DeclSyntax = "var _storage: [String: Any] = [:]"
+    storage.leadingTrivia = [.newlines(1), .spaces(2)]
+    return [storage]
```

Comment:
What about simplifying this further to

```suggestion
    return ["\n  var _storage: [String: Any] = [:]"]
```